components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2421,6 +2421,7 @@ void RewardsServiceImpl::OnNotificationTimerFired() {
   GetReconcileStamp(
       base::Bind(&RewardsServiceImpl::MaybeShowAddFundsNotification,
         AsWeakPtr()));
+  FetchGrants(std::string(), std::string());
```

Comment:
Note: When we just want to fetch all grants, we can just pass empty strings, as we do here:
https://github.com/brave/brave-core/blob/master/browser/extensions/api/brave_rewards_api.cc#L145